src/index.ts

```diff
@@ -1,142 +1,133 @@
+      phaseResolver,
+      teamResolver,
+      notificationResolver,
+      eventResolvers,
+      ticketResolver,
+      DocumentationResolvers,
+      attendanceResolver,
+      Sessionresolvers,
+    ]),
+  });
+
+  const pubsub = new PubSub();
   const server = new ApolloServer({
     schema,
     introspection: true,
-    // context: ({ req , res }:any) => ({ req, res, pubsub }),
     plugins: [
       ApolloServerPluginDrainHttpServer({ httpServer }),
       {
         async serverWillStart() {
           return {
             async drainServer() {
-              subscriptionServer.close()
+              subscriptionServer.close();
             },
-          }
+          };
         },
       },
+      logGraphQLRequests,
     ],
-    cache: 'bounded',
-    formatError: formatError,
-    context: context,
+    context,
+    formatError: (err) => {
+      // Log the error using tslog
+      logger.error(`${err}`);
+      return err;
+    },
     csrfPrevention: true,
-  })
+  });
 
-  const subscriptionServer = SubscriptionServer.create(
-    {
-      // This is the `schema` we just created.
-      schema,
-      // These are imported from `graphql`.
-      execute,
-      subscribe,
-      async onConnect() {
-        console.log('Connected!')
-        // If an object is returned here, it will be passed as the `context`
-        // argument to your subscription resolvers.
-        return {
-          pubsub,
-        }
-      },
-      onDisconnect() {
-        console.log('Disconnected!')
-      },
+  const subscriptionServer = SubscriptionServer.create({
+    schema,
+    execute,
+    subscribe,
+    onConnect() {
+      logger.info('Connected!');
+      return { pubsub };
     },
-    {
-      // This is the `httpServer` we created in a previous step.
-      server: httpServer,
-      // This `server` is the instance returned from `new ApolloServer`.
-      path: '/',
-    }
-  )
-
-  // Setup serving of static files or images.
-  app.use('/images', express.static('public'))
-
-  // More required logic for integrating with Express
-  await server.start()
-  server.applyMiddleware({
-    app,
+    onDisconnect() {
+      logger.warn('Disconnected!');
+    },
+  }, {
+    server: httpServer,
     path: '/',
-  })
+  });
+
+  app.use('/images', express.static('public'));
 
-  // Modified server startup
-  const PORT: number = parseInt(process.env.PORT!) || 4000
+  await server.start();
+  server.applyMiddleware({ app, path: '/' });
 
   connect().then(() => {
-    console.log('Database Connected')
-    httpServer.listen({ port: PORT }, () =>
-      console.log(
-        `🚀 Server ready at http://localhost:${PORT}${server.graphqlPath}`
-      )
-    )
-  })
-})(typeDefs, resolvers)
+    console.log('Database Connected.');
+    console.log(`Environment is set to ${process.env.NODE_ENV}`);
+    httpServer.listen({ port: PORT }, () => {
+      console.log(`🚀 Server ready at http://localhost:${PORT}${server.graphqlPath}`);
```

Comment:
Did you intentionally leave these as `console.log`?

Reply:
It's probably okay that way. There could be cases where we want to toggle the logger to off and we'd still want to see these.